packages/core/EventManager.ts

```diff
@@ -338,7 +341,25 @@ export default class EventManager {
   private async createAllCalendarEvents(event: CalendarEvent) {
     let createdEvents: EventResult<NewCalendarEventType>[] = [];
     if (event.destinationCalendar && event.destinationCalendar.length > 0) {
-      for (const destination of event.destinationCalendar) {
+      // Since GCal pushes events to multiple calendars we only want to create one event
+      let gCalAdded = false;
+      const destinationCalendars: DestinationCalendar[] = event.destinationCalendar.reduce(
+        (destintaionCals, cal) => {
+          if (cal.integration === "google_calendar") {
+            if (gCalAdded) {
+              return destintaionCals;
+            } else {
+              gCalAdded = true;
+              destintaionCals.push(cal);
+            }
+          } else {
+            destintaionCals.push(cal);
+          }
+          return destinationCalendars;
+        },
+        [] as DestinationCalendar[]
+      );
+      for (const destination of destinationCalendars) {
```

Comment:
@alannnc @emrysal filtering to only create one GCal event among all destination calendars. Other calendar integrations will still get an event per calendar.

Reply:
I still need to test this